rllib/policy/torch_policy_v2.py

```diff
@@ -803,8 +810,18 @@ def learn_on_loaded_batch(self, offset: int = 0, buffer_index: int = 0):
             )
             batch_fetches[f"tower_{i}"] = {"custom_metrics": custom_metrics}
 
+        # Determine whether we need to copy the batch to the device
```

Comment:
Actually, this comment is a little confusing.
Maybe change it (and the one above) to something like:
"If `_load_only_minibatch_onto_device` is True, then the main batch always remains on the CPU (it's probably too big to be fit on the GPU). Thus in this case, for each individual update step, we need to copy the freshly determined sub-slice to the GPU. These sub-slices need to be small enough then to fit on the GPU."